app/Models/User.php

```diff
@@ -26,7 +26,7 @@ class User extends Authenticatable
     /**
      * The attributes that should be hidden for serialization.
      *
-     * @var array<int, string>
+     * @var array<string>
```

Comment:
```suggestion
     * @var list<string>
```

`list` solves the issue with defaults of Larastan 3.0 to prevent the covariant issue you are describing.

 However, this pattern currently isn't consistent across the framework.

Reply:
thanks for pointing this out @ahinkle. yah, looks like very limited use in the FW so far, but maybe something we want to look into.